pkg/test/spacebindingrequest/spacebindingrequest.go

```diff
@@ -0,0 +1,66 @@
+package spacebindingrequest
+
+import (
+	"time"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	"github.com/gofrs/uuid"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+type Option func(spaceRequest *toolchainv1alpha1.SpaceBindingRequest)
+
+func NewSpaceBindingRequest(name, namespace string, options ...Option) *toolchainv1alpha1.SpaceBindingRequest {
+	spaceBindingRequest := &toolchainv1alpha1.SpaceBindingRequest{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+			UID:       types.UID(uuid.Must(uuid.NewV4()).String()),
```

Comment:
this can cause a `panic`. Is it ok or we'd prefer to return `(*toolchainv1alpha1.SpaceBindingRequest, error)`?

Reply:
I think it's fine to ignore that, since those are test utilities , not production code. But I'm fine with adding the err if you think it's more correct.